index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="index.css">
+</head>
+<body>
+    <div class="landingPage">
+        <header>
+            <div class="logo">My Logo</div>
+            <div class="menu">
+                <img src="./images/menu.svg" class="menuIcon" alt="">
+            </div>
+        </header>
+        <div class="container">
```

Comment:
- [ ] [OPTIONAL] In my opinion `section` tag is more semantic than the `div` tag in this position 👍🏼